include/overlay_50.h

```diff
@@ -0,0 +1,29 @@
+#ifndef POKEHEARTGOLD_OVERLAY_50_H
+#define POKEHEARTGOLD_OVERLAY_50_H
+
+#include "save.h"
+
+typedef struct {
+    u8 unk0[4];
+    u8 unk4a:4;
+    u8 unk4b:1;
+    u8 unk5[0x63];
+    u8 unk68[0x600];
+    u8 unk668[6];
+    SAVEDATA *saveData;
+    u32 unk674;
+} Unk021E5A3C;
```

Comment:
If you're using hex addresses for structs defined in overlay modules, you should also specify which overlay they are tied to. Reduces the chance of namespace collisions.

Reply:
Done